README.md

```diff
@@ -22,18 +22,21 @@ How to initialise your configuration
 6) [Profile Retrieval](#profile-retrieval) -
 How to retrieve a Yoti profile using the token
 
-7) [Handling Users](#handling-users) -
+7) [Dynamic Policy Generation](#dynamic-policy-generation) -
```

Comment:
I would title this something like `Programmatic QR code creation` as `policy` is not a concept that is exposed at an SDK level.